src/packages/auth/functions.ts

```diff
@@ -260,3 +262,52 @@ export async function user(authorizationHeader: string | undefined): Promise<Use
     };
   }
 }
+
+export async function changeRole(
+  authorizationHeader: string | undefined,
+  role: string | undefined,
+): Promise<AuthResponse> {
+  const { AUTH_TYPE } = getEnv();
+  const jwtPayload = decodeJwt(authorizationHeader);
```

Comment:
Instead of decode, can we verify? Just to make sure the given token is not forged.